Maskbook/Scene/App/FIleService/Model/FileServiceUploadOption.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-struct FileServiceUploadOption {
+struct FileServiceUploadOption: Hashable {
```

Comment:
Leave `FileServiceUploadOption` alone, you don't need to change it.

Reply:
ok